### DISCUSSION TITLE
Forecast Temperature as QuantitativeValue

### DISCUSSION THREAD

**Discussion by aneugart** [weather-gov/api#488] [Q&A]
I have a few questions about the forecast_temperature_qv and forecast_wind_speed_qv feature flags on the .../forecast endpoint.  

Previously when I have used the .../forecast endpoint setting the units argument to "si" or "us" would change both the units for the individual values and the units of the measurements included in the detailedForecast.  Now, with the use of the "forecast_temperature_qv" feature flag the units for the individual values remain in SI units regardless of the value of the units argument.  The measurements in the detailedForecast still respect the value of the units argument.  Is this the new intended behavior or is this subject to change?

Also, previously the forecasted temperature values were rounded to the nearest integer value whereas with these feature flags the values are not.  Is this the new intended behavior?

Finally, I was wondering if it is known when these changes will become the default behavior, I have not noticed a SCN on this.

Thanks for the help!

**Comment by pettijohn** [comment]
The docs do say "Use US customary or SI (metric) units in textual output," so I guess you're right that, since QV is not textual output, the units request does not work. Wish it did!

**Reply by dwhitemv25** [reply]
Please be careful about necro-ing old posts, the original post was from 2021.

Based on https://github.com/weather-gov/api/discussions/394#discussioncomment-592878 (from 2019!), QV is designed to be easily machine-parsed, so just convert it to your preferred unit. I suspect the extra precision was another driver for this change, as the original API defined the field as an integer.
